clickhouse_driver/client.py

```diff
@@ -567,6 +567,9 @@ def from_url(cls, url):
             elif name == 'compress_block_size':
                 kwargs[name] = int(value)
 
+            elif name == 'settings_is_important':
+                kwargs[name] = int(value)
```

Comment:
It'd be better to make this parameter boolean like `verify` option below.

Reply:
Ok

Reply:
Done